tests/units/units.cpp

```diff
@@ -2731,3 +2731,28 @@ TEST(Units, circularImportDeeperLevelBaseUnits)
 
     EXPECT_FALSE(units1->isBaseUnit());
 }
+
+TEST(Units, coverage)
+{
+    libcellml::UnitsItemPtr unitsItem;
+
+    {
+        auto units = libcellml::Units::create("units");
+
+        units->addUnit("second", 1.0, "second_id");
+
+        auto model = libcellml::Model::create("model");
+
+        model->addUnits(units);
+
+        auto annotator = libcellml::Annotator::create();
+
+        annotator->setModel(model);
+
+        unitsItem = annotator->item("second_id")->unitsItem();
+
+        EXPECT_TRUE(unitsItem->isValid());
+    }
+
+    EXPECT_FALSE(unitsItem->isValid());
+}
```

Comment:
If this is purely for coverage reasons then it belongs in the coverage test.

Reply:
FWIW, this test has been superseded by commit 157189f5. However, it has now been moved to the coverage test file (see commit 172ae199).